newton/examples/elastic_rod/rod_mesher.py

```
@@ -0,0 +1,412 @@
+        device: wp.Device | None = None,
+    ):
+        self.num_rods = num_rods
+        self.num_points = num_points
+        self.radius = radius
+        self.resolution = resolution
+        self.smoothing = smoothing
+        self.device = device or wp.get_device()
+
+        self.num_rings = (num_points - 1) * smoothing + 1 if num_points >= 2 else 0
+        verts_per_rod = self.num_rings * resolution
+        tris_per_rod = (self.num_rings - 1) * resolution * 2 if self.num_rings > 1 else 0
```

Ruff (0.15.4)

[error] Local variable `tris_per_rod` is assigned to but never used

Remove assignment to unused variable `tris_per_rod`

(F841)